cmd/subnet-utils.go

```diff
@@ -782,9 +782,10 @@ func getAPIKeyFlag(ctx *cli.Context) (string, error) {
 	return apiKey, nil
 }
 
-func initSubnetConnectivity(ctx *cli.Context, aliasedURL string, forUpload bool, failOnConnErr bool) (string, string) {
-	e := validateSubnetFlags(ctx, forUpload)
-	fatalIf(probe.NewError(e), "Invalid flags:")
+func initSubnetConnectivity(ctx *cli.Context, aliasedURL string, failOnConnErr bool) (string, string) {
+	if ctx.IsSet("airgap") && len(ctx.String("api-key")) > 0 {
```

Comment:
The problem with is set is that, `--airgap=false`  will give ctx.IsSet("airgap') == true. ctx.Bool() is better

Reply:
@vadmeste thanks, fixed.